### PR TITLE
EZP-28567: As an editor I want to be able to restore archived Content version

### PR DESCRIPTION
**Pull request by webhdx**
| Question      | Answer
| ------------- | ---
| Tickets       | https://jira.ez.no/browse/EZP-28567
| Bug fix?      |no
| New feature?  | yes
| BC breaks?    | no
| Tests pass?   | yes
| Doc needed?   | no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

#### Checklist:
- [ ] Coding standards (`$ composer fix-cs`)
- [ ] Ready for Code Review

### CODE REVIEW

File: src/bundle/Resources/views/content/tab/versions/table.html.twig [hunk 1]
```diff
@@ -52,6 +53,15 @@
                     </a>
                 </td>
             {% endif %}
+            {% if is_archived %}
+                <td>
+                    <button class="btn btn-icon btn--restore-archived-version" data-content-id="{{ version.contentInfo.id }}" data-version-no="{{ version.versionNo }}" data-language-code="{{ version.initialLanguageCode }}">
```
Comment (dew326) [inline src/bundle/Resources/views/content/tab/versions/table.html.twig]: `ez-btn--restore-archived-version`

File: src/bundle/Resources/public/js/scripts/admin.location.version.restore_archived.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+(function (global, doc) {
+    const FORM_ARCHIVED_VERSION_RESTORE = 'form[name="archived_version_restore"]';
+
+    const restoreArchivedVersion = event => {
```
Comment (dew326) [inline src/bundle/Resources/public/js/scripts/admin.location.version.restore_archived.js]: `(event)`

File: src/bundle/Resources/public/js/scripts/admin.location.version.restore_archived.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
+(function (global, doc) {
+    const FORM_ARCHIVED_VERSION_RESTORE = 'form[name="archived_version_restore"]';
+
```
Comment (dew326) [inline src/bundle/Resources/public/js/scripts/admin.location.version.restore_archived.js]: remove empty line

**Comment by webhdx** [comment]
@dew326 Fixed.

**Pull request merged**